.github/workflows/daily-deploy-production.yml

```diff
@@ -36,7 +36,19 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-
+          
+      - name: Install dependencies
+        uses: ./.github/workflows/install
+        with:
+          key: ${{ hashFiles('yarn.lock') }}
+          yarn_cache_folder: .cache/yarn
+          path: |
+            .cache/yarn
+            node_modules
+      
+      - name: Build
+        run: yarn build --verbose —buildtype=vagovprod
+        timeout-minutes: 30
```

Comment:
This actually isn't changing the functionality of the daily deploy, but rather just building prod before the current process executes. 

We would need to update the [Deploy step](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/workflows/daily-deploy-production.yml#L186-L191) in the `Deploy` job to upload the assets from this build to the prod S3 buckets rather than still moving over the assets in the latest commit's bucket to the prod bucket.

Also, I think we still want to validate the build status before we run the build, so that way we aren't running the full build just for the deploy to fail if the build status isn't successful